src/DynamoCoreWpf/DynamoCoreWpf.csproj

```diff
@@ -111,6 +111,7 @@
     <Compile Include="Extensions\ViewExtensionDefinition.cs" />
     <Compile Include="Extensions\ViewStartupParams.cs" />
     <Compile Include="Interfaces\IShellCom.cs" />
+    <Compile Include="Interfaces\IWorkspaceViewModel.cs" />
```

Comment:
Let's remove `IWorkspaceViewModel` as described below.  Clients can get access to `IWorkspaceModel` through the `ViewLoadedParams` object as it will subclass `ReadyParams`, which should now have a `WorkspaceModels` property.